Gopkg.lock

```diff
@@ -1680,13 +2015,15 @@
     "util/integer",
     "util/jsonpath",
     "util/retry",
-    "util/workqueue"
+    "util/workqueue",
   ]
+  pruneopts = "NUT"
   revision = "23781f4d6632d88e869066eaebb743857aa1ef9b"
   version = "v7.0.0"
 
 [[projects]]
   branch = "release-2.9"
```

Comment:
It looks like helm version didn't change.

Reply:
It's my bad I mixed up something at the dep update gona fix